package.json

```diff
@@ -12,7 +12,7 @@
   "devDependencies": {
     "autoprefixer": "^7.1.3",
     "cookie-policy": "^0.0.1",
-    "global-nav": "^0.1.3",
+    "global-nav": "^0.1.4",
```

Comment:
Was this published to npm? Looks like 0.1.3 is the latest version: https://www.npmjs.com/package/global-nav